engine/topology/src/main/java/org/opennms/oce/engine/topology/model/ModelBuilderImpl.java

```diff
@@ -38,28 +38,32 @@
 import org.opennms.oce.datasource.api.InventoryObjectPeerRef;
 import org.opennms.oce.datasource.api.InventoryObjectRelativeRef;
 
+/**
+ * the class is deprecated now and replaced with stateful InventoryModelManager
+ */
+@Deprecated
```

Comment:
Is there any reason to keep this around rather than just migrating everything to use the new `InventoryModelManager`?